themes/radial/components/graphs.typ

```diff
@@ -0,0 +1,94 @@
+#import "../colors.typ": *
+
+#import "@preview/cetz:0.1.2"
+
+/// Creates a labeled pie chart.
+///
+/// Example Usage:
+///
+/// ```typ
+/// #pie_chart(
+///   (8, green, "wins"),
+///   (2, red, "losses")
+/// )
+/// ```
+///
+/// - ..data (array): The first field is the value, the second the color, and the third is the name of the data.
```

Comment:
Specify what data type the function will expect each variable in the array to be.